attachments/tests/base.py

```diff
@@ -41,10 +41,11 @@ def setUp(self):
 
         self.obj = TestModel.objects.create(title="My first test item")
 
-    def _upload_testfile(self, file_obj=None):
+    def _upload_testfile(self, file_obj=None, opts=None):
         """
         Uploads a sample file for the given user.
         """
+        opts = {} if opts == None else opts
```

Comment:
Comparisons with `None` need to use `is`.
```suggestion
        opts = opts or {}
```